### PR TITLE
Rephrase comment on BigInt operators

### PR DESCRIPTION
**Pull request by webstrand**
Got rid of the "also unsupported" phrasing and place more emphasis on `unary`

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/bigint/index.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ The following operators may be used with BigInt values or object-wrapped BigInt
 + * - % **
 ```
 
-[Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators) are supported as well, except `>>>` (zero-fill right shift), as every BigInt value is signed.
+However, the unary operator (`+`) cannot be supported on BigInt due to conflicting usage in asm.js, so it has been omitted [in order to not break asm.js](https://github.com/tc39/proposal-bigint/blob/master/ADVANCED.md#dont-break-asmjs).
 
-Also unsupported is the unary operator (`+`), [in order to not break asm.js](https://github.com/tc39/proposal-bigint/blob/master/ADVANCED.md#dont-break-asmjs).
+All of the [Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators) are supported as well, except for `>>>` (zero-fill right shift), as every BigInt value is signed.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/bigint/index.md:64]: I suggest rewording this entire section, so it reads like:

```md
Most operators accept BigInt operands. Some of them involving number logic include:

- [Increment and decrement](/en-US/docs/Web/JavaScript/Reference/Operators#increment_and_decrement): `++`, `--`
- [Unary negation (`-`)](/en-US/docs/Web/JavaScript/Reference/Operators/Unary_negation)
- [Arithmetic operators](/en-US/docs/Web/JavaScript/Reference/Operators#arithmetic_operators): `+`, `-`, `*`, `/`, `%`, `**`
- [Relational operators](/en-US/docs/Web/JavaScript/Reference/Operators#relational_operators) and [Equality operators](/en-US/docs/Web/JavaScript/Reference/Operators#equality_operators): `>`, `<`, `>=`, `<=`, `==`, `!=`, `===`, `!==`
- Unary `-`
- [Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators#bitwise_shift_operators): `>>`, `<<`, `&`, `|`, `^`, `~`

Others, such as [logical operators](/en-US/docs/Web/JavaScript/Reference/Operators#binary_logical_operators), only use the truthiness of operands.

There are the following caveats:

- The unary operator (`+`) cannot be supported on BigInt due to conflicting usage in asm.js, so it has been omitted [in order to not break asm.js](https://github.com/tc39/proposal-bigint/blob/master/ADVANCED.md#dont-break-asmjs).
- Unsigned right shift (`>>>`) is the only bitwise operator that's unsupported, as every BigInt value is signed.
- All binary operators must have numbers or BigInts on both sides — they cannot be mixed. The only exceptions are the equality and relational operators — you can compare numbers with BigInts.
- The `/` operator works as expected with whole numbers, but operations with a fractional result will be truncated without returning any fractional digits.
```

The exact wording is up to you.

Comment (webstrand) [reply files/en-us/web/javascript/reference/global_objects/bigint/index.md]: I've rewritten the entire section, I believe I've gotten everything you mentioned.

File: files/en-us/web/javascript/reference/global_objects/bigint/index.md [hunk 2]
```diff
@@ -53,50 +53,39 @@ typeof Object(1n) === "object"; // true
...
-const maxPlusOne = previousMaxSafe + 1n;
-// 9007199254740992n
+- [Arithmetic operators](/en-US/docs/Web/JavaScript/Reference/Operators#arithmetic_operators): `+`, `-`, `*`, `/`, `%`, `**`
+- [Bitwise operators](/en-US/docs/Web/JavaScript/Reference/Operators#bitwise_shift_operators): `>>`, `<<`, `&`, `|`, `^`, `~`
+- [Unary negation (`-`)](/en-US/docs/Web/JavaScript/Reference/Operators/Unary_negation)
+- [Increment/decrement](/en-US/docs/Web/JavaScript/Reference/Operators#increment_and_decrement): `++`, `--`
 
-const theFuture = previousMaxSafe + 2n;
-// 9007199254740993n, this works now!
+Mixed operand types are allowed by the boolean-returning operators:
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/bigint/index.md:63]:
```suggestion
The boolean-returning operators allow mixing numbers and BigInts as operands: 
```

**Review by Josh-Cena — APPROVED** [review]
LGTM. Made a few small edits. Thanks and you are part of MDN now, webstrand ;)
